### DISCUSSION TITLE
The Binding of Isaac: Afterbirth+

### DISCUSSION THREAD

**Discussion by Jeff-2612** [skyline-emu/skyline#1192] [Game Requests]
they are saying that The Binding of Isaac: Afterbirth+ is playable, but I think they are testing the Rebirth version. Will they be able to fix this problem in the future?

**Comment by PixelyIon** [comment]
It is actually Afterbirth+ but it's only possible with manually patching the ROM to hardcode the DLC which is a fairly involved process that I won't cover in here, we will add DLC/Update support in the future so this won't be required.

**Reply by Jeff-2612** [reply]
Ok thanks, can you point me to a tutorial or research doing this hand coding so I can do it here?

**Reply by projsMED** [reply]
Do you know if it's possible for me to upgrade using Yuzu and share the upgraded room to Skyline?

**Reply by PixelyIon** [reply]
It isn't, the process is really complicated and you should just wait for eventual DLC/Update support.

**Reply by Jeff-2612** [reply]
Allright thanks.
